src/print.jl

```diff
@@ -282,6 +282,24 @@ function show_backend_summary(io::IO, model::GenericModel)
     return
 end
 
+"""
+    const _CONSTRAINT_LIMIT_FOR_PRINTING = Ref{Int}(100)
+
+A global constant used to control when constraints are omitted when printing
+the model.
+
+Get and set this value using `_CONSTRAINT_LIMIT_FOR_PRINTING[]`.
+
+```julia
+julia> _CONSTRAINT_LIMIT_FOR_PRINTING[]
+100
+
+julia> _CONSTRAINT_LIMIT_FOR_PRINTING[] = 10
+10
+```
+"""
+const _CONSTRAINT_LIMIT_FOR_PRINTING = Ref{Int}(100)
```

Comment:
Is it too bad if it is a field in the model struct?

Reply:
That would be an alternative. I'm open to discussion more generally on the right approach for this. I don't particularly like the way it currently is.